tasks/setup_plugins.yml

```diff
@@ -21,6 +21,8 @@
     update: True
   tags: [ 'role::checkmk_agent:plugins:get' ]
   delegate_to: '{{ checkmk_agent__git_dest_host }}'
+  become: '{{ False if (checkmk_agent__git_dest_host in ["localhost"]) else True }}'
```

Comment:
I think that extending the list wouldn't make sense here or then the list would become a variable anyway. So please change to a simple string comparison, yes.